### ISSUE TITLE
Sync saved places/locations with uMap/cloud CSV

### ISSUE THREAD

**Issue by xplosionmind** [organicmaps/organicmaps#1082]
I have [a public map of saved locations](https://umap.openstreetmap.fr/en/map/favorites_593427#8/44.922/10.651 'Tommi’s Favolrites') with [uMap](https://umap.openstreetmap.fr 'uMap').

Now, it would be awesome if my list of saved locations could be synchronized between my Orcanic Maps (iOS) app and my personal uMap.

Another great alternative would be to synchronize saved locations in a CSV which could be kept up-to-date in a cloud service. For example, I have a cloud folder synchronized with [data](https://github.com/xplosionmind/data 'Tommi’s data repository on GitHub')  in which you can find [`places.csv`](https://github.com/xplosionmind/data/blob/main/places.csv 'Tommi’s favorite places collected in this CSV file'); Organic Maps could link my saved locations to that CSV and automatically update it when I make changes in the app or, vice-versa, when I edit the CSV.

**Comment by rtsisyk** [comment]
Yes, this is a good idea. Moreover, we have our own uMap installation for testing (https://web.organicmaps.app).

**Comment by keunes** [comment]
Related (re sync): #622
